### PR TITLE
Update system-requirements.md

### PR DESCRIPTION
**Pull request by aalejandroaraujo**
While supporting a customer in the migration process from Lync Server 2013 to Skype for Business Server 2019 we've received a crystal clear popup showing that, in order to be able to run Setup.exe .NET Framework 4.7 was required, hence made the change in the article.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfBServer2019/plan/system-requirements.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ There are some things you're going to need to install or configure for any serve
...
+|Microsoft .NET Framework  <br/> |WCF services is a **Feature** that's installed as a Windows feature, under **Server Manager**, initially no downloads needed. <br/> • You need to make sure, when you install this feature, or if it's already installed and you're checking on it, that the **HTTP Activation** option is also checked and installed, like so: <br/> ![Screenshot showing HTTP Activation option under the .NET Framework 4.5 Features.](../../SfbServer/media/a4064fa0-fa49-4474-bd98-b9a79ff68f8b.png) <br/> Don't worry if you get an additional pop-up saying some other things need to be installed for HTTP Activation to be installed. That's normal; click OK and go ahead. If you don't get this pop-up, you can assume those things are already installed and go ahead.  <br/> Microsoft .NET Framework is usually installed when Windows Server 2016 is installed. Skype for Business Server requires Microsoft .NET Framework 4.7 though, so you'd probably need to update it. You can find the update [here](https://support.microsoft.com/en-us/help/3186497/the-net-framework-4-7-offline-installer-for-windows/)<br/> |
```
Comment (nenonix) [inline Skype/SfBServer2019/plan/system-requirements.md]:
```suggestion
|Microsoft .NET Framework  <br/> |WCF services is a **Feature** that's installed as a Windows feature, under **Server Manager**, initially no downloads needed. <br/> • You need to make sure, when you install this feature, or if it's already installed and you're checking on it, that the **HTTP Activation** option is also checked and installed, like so: <br/> ![Screenshot showing HTTP Activation option under the .NET Framework 4.5 Features.](../../SfbServer/media/a4064fa0-fa49-4474-bd98-b9a79ff68f8b.png) <br/> Don't worry if you get an additional pop-up saying some other things need to be installed for HTTP Activation to be installed. That's normal; click OK and go ahead. If you don't get this pop-up, you can assume those things are already installed and go ahead.  <br/> Microsoft .NET Framework is usually installed when Windows Server 2016 is installed. Skype for Business Server requires Microsoft .NET Framework 4.7 though, so you probably need to update it. You can find the update [here](https://support.microsoft.com/help/3186497/the-net-framework-4-7-offline-installer-for-windows/)<br/> |
```

**Comment by scanum** [comment]
@MicrosoftHeidi Hi! PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@MicrosoftHeidi Hi Heidi, any updates on this? Thanks!

**Comment by MicrosoftHeidi** [comment]
This has been confirmed as correct by the PG.

**Pull request merged**